python/setup.py

```diff
@@ -429,44 +393,12 @@ def HasLibraryDirsOpt():
     ])
     os.environ['PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION'] = 'cpp'
 
-  # Keep this list of dependencies in sync with tox.ini.
-  install_requires = []
-
   setup(
-      name='protobuf',
-      version=GetVersion(),
-      description='Protocol Buffers',
-      download_url='https://github.com/protocolbuffers/protobuf/releases',
-      long_description="Protocol Buffers are Google's data interchange format",
-      url='https://developers.google.com/protocol-buffers/',
-      project_urls={
-          'Source': 'https://github.com/protocolbuffers/protobuf',
-      },
-      maintainer='protobuf@googlegroups.com',
-      maintainer_email='protobuf@googlegroups.com',
-      license='BSD-3-Clause',
-      classifiers=[
-          'Programming Language :: Python',
-          'Programming Language :: Python :: 3',
-          'Programming Language :: Python :: 3.7',
-          'Programming Language :: Python :: 3.8',
-          'Programming Language :: Python :: 3.9',
-          'Programming Language :: Python :: 3.10',
-      ],
-      namespace_packages=['google'],
-      packages=find_packages(
-          exclude=[
-              'import_test_package',
-              'protobuf_distutils',
-          ],),
       test_suite='google.protobuf.internal',
       cmdclass={
-          'clean': CleanCmd,
```

Comment:
why this is removed?

Reply:
Hi Jie, this setup by default will clean up some type of build-time only artefacts, especially applicable to those defined in the CleanCmd by their file extension. I’d be happy to hear your feedback further.